code/modules/jobs/job_types/roguetown/garrison/citywatch.dm

```diff
@@ -45,27 +45,22 @@
 	ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
 
 /datum/outfit/job/roguetown/citywatch/proc/assign_skills(mob/living/carbon/human/guard)
-	guard.mind.adjust_skillrank(/datum/skill/combat/maces, pick(3,4), TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/unarmed, 4, TRUE) 
-	guard.mind.adjust_skillrank(/datum/skill/combat/swords, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 3, TRUE)
```

Comment:
they are meant to be somewhat trained in all arms, I would suggest giving them whips&flails back, but it could be interesting not to.